### ISSUE TITLE
Show tables and plots side-by-side?

### ISSUE THREAD

**Issue by jhofman** [jhofman/datamations#30]
@dggoldst and I realized that there are some data operations that make sense in table-based datamations but not in plot-based ones and vice-versa.

For instance, consider the "select" operation where only certain columns are retained. It's not clear how this would be represented in a plot, but in a table you'd simply drop the columns.

Maybe one way out of this is that we show table and plots side-by-side, and some animations affect one but not the other. We could also add interactivity so that when you hover over an observation in the plot it higlights in the table and vice-versa.

**Comment by jhofman** [comment]
@sharlagelfand just reviving this a bit.

do you think there's a simple drop-in thing here where we could use something like [DT](https://rstudio.github.io/DT/) to show dataframes as tables alongside the animations?

for instance, each step in the pipeline could be on a different tab, with the tab showing the results at that point?

if it's easy to play with this for some brainstorming, great. from there we can think about how to synchronize that with the animation.

but if it's a pain let's not dig too deep into it just yet.

**Comment by sharlagelfand** [comment]
@jhofman yep definitely! having each of the results in a tab is a great idea. I'll play around with how it could look/work.

**Comment by sharlagelfand** [comment]
Have a (somewhat buggy) proof of concept [here](https://sharlagelfand.shinyapps.io/datamations/) showing each step in a tab - I've limited it to one tab for each step, so just one for all "group by" rather than one for each variable grouped.

Initial data shows all of the data, all variables, and in whatever order it comes

Group by shows just the grouping variables and the summary variable, and the data is arranged by the grouping variable

Summary data shows the summarised data.

We may not want to show _all the data_ in the first tab (are all of the columns relevant?) but I think having it gives a better distinction between the "initial data" and "group by" tabs, especially apparent when looking at the penguins data!

**Comment by jhofman** [comment]
great first pass on this. let's discuss at our next meeting.

**Comment by sharlagelfand** [comment]
Closing this, #56 covers next steps re: using slider to change tab.
